tweets.py

```diff
@@ -1,25 +1,33 @@
 import requests
 import csv
 from pandas import Timestamp, Timedelta
-from utils import list_average
+from utils import list_average, get_reader
 
 DATA_URL = 'https://www.istheweatherweird.com/istheweatherweird-data-hourly'
 STATIONS_URL = '{}/csv/stations.csv'.format(DATA_URL)
 
-CITY = 'Chicago'
 START_TIME = Timestamp.utcnow().replace(microsecond=0) - Timedelta(days=1)
 END_TIME = Timestamp.utcnow().replace(microsecond=0)
```

Comment:
For a couple of reasons I think it would be better to decalre the start and end times in `get_tweets()` and then pass them to `get_observations()` and `get_historical_temps()` as needed

Reply:
Also if by the time this code gets executed its 6:01pm then will you have a slight problem? Why not just set `END_TIME` to exactly 6PM local time? And use a different variable `current_time` to make the decision about whether to tweet.

Reply:
keeping `start_time`, `end_time`, and `current_time` separate makes sense to me